auth/ipa_ad_trust.adoc

```diff
@@ -28,11 +28,24 @@ The following is needed in order to enable AD Trust Authentication to the Applia
 [[configure-ipa-ad-trust-server]]
 == Set up and configure an IPA server for AD Trust Authentication
 
-* IPA Server based on FreeIPA 3.3.3 or later is recommended
+* Configure an IPA Server based on FreeIPA 3.3.3 or later
 
 Instructions for setting up and configuring cross-realm trust between an IPA domain and an AD (Active Directory) domain
 can be found at http://www.freeipa.org/page/Active_Directory_trust_setup[freeipa.org Active Directory Trust Setup]
 
+* Add necessary user attributes to the SSSD Configuration on the IPA Server
```

Comment:
A couple tiny edits:
- lowercase for "configuration" and "server" in "SSSD Configuration on the IPA Server"
- this file path should be marked up with backticks  -->   `/etc/sssd/sssd.conf`